elastic/kibana
# [ECO] Use new Entity v2 endpoints in Inventory

## Summary

Closes https://github.com/elastic/kibana/issues/202295
Closes https://github.com/elastic/kibana/issues/202296

This PR adapts Inventory to use the new Entity v2 endpoints.

## Testing
- Use any synthtrace scenario that loads service/hosts/containers data
- Navigate and make sure everything works as expected (navigation to Discovery/Infra/Services pages, interacting with the table, searching for some specific entity, interacting with the type filter)
- To check the alerts work, it's easier to connect to a remote cluster.
